spec/decorators/user_decorator_spec.rb

```diff
@@ -72,6 +72,97 @@
     end
   end
 
+  describe '#pending_profile_requires_verification?' do
+    it 'returns false when no pending profile exists' do
+      profile = create(:profile)
+      user_decorator = UserDecorator.new(profile.user)
+
+      expect(user_decorator.pending_profile_requires_verification?).to eq false
+    end
+
+    it 'returns true when no active profile exists' do
```

Comment:
This spec title confused me at first. I thought the user did not have any profiles at all. Since there are 3 different reasons why a profile can be deactivated, how about a more specific title like "returns true when the user has a deactivated profile due to pending verification"? 

And then please add 2 more tests to verify that if the deactivated_reason is `password_reset` or `encryption_error`, then this returns false.